lib/jsonapi/query_builder/mixins/sort.rb

```diff
@@ -20,6 +22,10 @@ def supported_sorts
             @supported_sorts || {}
           end
 
+          def supported_dynamic_sorts
+            @supported_dynamic_sorts || []
```

Comment:
Combined with the next suggestion (ref: 1)
```suggestion
            @supported_dynamic_sorts ||= []
```